PostBuild.sh

```diff
@@ -8,6 +8,7 @@ PROGNAME=$(basename "$0")
 CHROOTMNT="${CHROOT:-/mnt/ec2-root}"
 DEBUG="${DEBUG:-UNDEF}"
 FIPSDISABLE="${FIPSDISABLE:-UNDEF}"
+GRUBTMOUT="${GRUBTMOUT:-5}"
```

Comment:
Did you mean to change the "default" from 0 to 5?

Reply:
Yes.